src/plugins/intel_cpu/src/config.h

```diff
@@ -65,6 +65,16 @@ struct Config {
     // std::hash<int> is necessary for Ubuntu-16.04 (gcc-5.4 and defect in C++11 standart)
     std::unordered_map<FILTER, std::string, std::hash<int>> blobDumpFilters;
 
+    enum FUSING_FILTER {
+        BY_FUSER_TYPE,
+        BY_FUSEE_TYPE,
+        BY_FUSER_NAME,
+        BY_FUSEE_NAME,
+    };
+
+    // std::hash<int> is necessary for Ubuntu-16.04 (gcc-5.4 and defect in C++11 standart)
+    std::unordered_map<FUSING_FILTER, std::string, std::hash<int>> fusingFilters;
```

Comment:
explicitly typed enum won't work without explicit hash function
I will try underlying_type and see if it works

Reply:
I mean you can explicitly set `FUSING_FILTER` enum type as int so that `std::hash<int>` will be always relevant, but  using `std::underlying_type` is more reliable approach

Reply:
replaced with underlying_type